src/main.rs

```diff
@@ -80,6 +85,7 @@ struct Args {
     #[clap(short, long, env = "ROCKET_ADDRESS", default_value = "::")]
     addr: IpAddr,
 
+    #[cfg(debug_assertions)]
     #[clap(short, long, env = "RENDER_EXTERNAL_HOSTNAME")]
```

Comment:
It's used to generate keys, which we wouldn't do in a production environment.

Reply:
Additionally, some tests were disabled for release mode (since Nix seems to test in release mode) due to this discovered bug https://github.com/profianinc/steward/issues/73